src/acceptance-test/java/org/zalando/nakadi/webservice/SettingsControllerAT.java

```diff
@@ -59,38 +50,17 @@ public void testUnblockFlooder() throws Exception {
     public void testGetFlooders() throws Exception {
         final EventType eventType = NakadiTestUtils.createEventType();
         blacklist(eventType.getName(), BlacklistService.Type.CONSUMER_ET);
-
-        TestUtils.waitFor(() -> given()
-                .contentType(ContentType.JSON)
-                .get(BLACKLIST_URL)
-                .then()
-                .statusCode(HttpStatus.SC_OK)
-                .content(getFloodersMatcher(eventType)), 1000, 200);
-
-    }
-
-    private SameJSONAs<? super String> getFloodersMatcher(final EventType eventType) {
-        try {
-            return JSON_HELPER.matchesObject(
-                    ImmutableMap.of(
-                            "consumers",  ImmutableMap.of(
-                                    "event_types", Collections.singleton(eventType.getName()),
-                                    "apps", Collections.emptySet()),
-                            "producers", ImmutableMap.of(
-                                    "event_types", Collections.emptySet(),
-                                    "apps", Collections.emptySet()))
-                    );
-        } catch (JsonProcessingException e) {
-            throw new RuntimeException();
-        }
-    }
-
-    private void clearFloodersData() {
         try {
-            CURATOR.delete().deletingChildrenIfNeeded().forPath("/nakadi/blacklist/consumers");
-            CURATOR.delete().deletingChildrenIfNeeded().forPath("/nakadi/blacklist/producers");
-        } catch (final Exception exception) {
-            // nothing to do
+            TestUtils.waitFor(
+                    () -> given()
+                            .contentType(ContentType.JSON)
+                            .get(BLACKLIST_URL)
+                            .then()
+                            .statusCode(HttpStatus.SC_OK)
+                            .body("consumers.event_types", hasItems(eventType.getName())),
+                    1000, 200);
+        } finally {
+            whitelist(eventType.getName(), BlacklistService.Type.CONSUMER_ET);
```

Comment:
@v-stepanov As a general rule with shared data if i created it, then I will delete it